avm/res/network/network-watcher/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,168 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'NetworkWatcherRG' // Note, this is the default NetworkWatcher resource group. Do not change.
+
+@description('Optional. The location to deploy resources to.')
+param resourcelocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'nnwmax'
+
```

Comment:
```suggestion
```

Reply:
`serviceShort` not required anymore? Which would mean we remove it everywhere?

Reply:
Hold your horses sir, no 😄 There are just 2 empty lines stacked on top of one another.

Reply:
I should stop using the PR comment extension in vscode ...